mycity/mycity/test/intent_base_case.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+
+import mycity.intents.intent_constants as intent_constants
+import mycity.mycity_controller as mcc
+import mycity.mycity_request_data_model as req
+import mycity.test.test_constants as test_constants
+
+
+
+
+###############################################################################
+# TestCase parent class for all intent TestCases, which are integration tests #
+# to see if any changes in codebase have broken response-request model.       #
+#                                                                             #
+# NOTE: Assumes that address has already been set.                            #
+###############################################################################
+
+class IntentBaseCase(unittest.TestCase):
+
+    __test__ = False
```

Comment:
Not sure if this is necessary given how we're loading the tests. It might be necessary if we load tests using a test discovery function